buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -34,6 +34,10 @@
 
 REPO_PATTERN = re.compile(r"https?://github.com/(?P<owner>[^/]+)/(?P<repo>[^/]+).git")
 
+EMOJI_PATTERN = re.compile(r":([^:\s]+):")
```

Comment:
I think we could use the regex specified in the [Buildkite docs](https://buildkite.com/docs/apis/rest-api/emojis) here:

```
re.compile(r":([\w+-]+):")
```